_data/simple-icons.json

```diff
@@ -5692,6 +5692,11 @@
             "hex": "1273EB",
             "source": "https://commons.wikimedia.org/wiki/File:Freepik.svg"
         },
+        {
+            "title": "FreeTube",
+            "hex": "000000",
+            "source": "https://raw.githubusercontent.com/FreeTubeApp/FreeTube/development/_icons/iconBlackSmall.svg"
```

Comment:
As per https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines, GitHub assets shouldn't be "raw" links. It should be a GitHub permalink - details available in the source guidelines section I just linked.

Reply:
Fixed it.
https://github.com/simple-icons/simple-icons/pull/11334/commits/e5f8b3d7c112a195055a31cf0cc1cb09467a442a

Reply:
Make sure you get a *permalink* of the file, where the commit hash is pinned in the URL. Instructions available in the link above.